src/index.js

```diff
@@ -1,7 +1,9 @@
 /* eslint-disable no-unused-vars */
 import _, { add } from 'lodash';
 import './style.css';
-import recentSores from './modules/setup.js';
+import {
+  gameScore, button, addScore, submitButton,
+} from './modules/hit-api.js';
 /* eslint-enable no-unused-vars */
```

Comment:
- [ ] Kindly remove the linters disabling here. Linters help us write standardized and maintain code